crates/sui-graphql-rpc/tests/snapshots/snapshot_tests__staging.graphql.snap

```diff
@@ -4393,7 +4393,7 @@ input TransactionBlockFilter {
 	inputObject: SuiAddress
 	"""
 	Limit to transactions that output a versioon of this object. NOTE: this input filter has
-	been deprecated in favor of `affectedObject` which offers an easier to understand behavor.
+	been deprecated in favor of `affectedObject` which offers an easier to understand behavior.
 	
 	This filter will be removed with 1.36.0 (2024-10-14), or at least one release after
```

Comment:
I'm guessing this is still needed since there's content. Apologies. I'll quit commenting on this.